.github/workflows/build.yml

```
@@ -158,6 +173,10 @@ jobs:
       - diagnostics
     runs-on: ubuntu-latest
```

Just to be consistent with other jobs that use `actions/checkout`.

```suggestion
    permissions:
      # actions/checkout needs this to fetch code
      contents: read
    runs-on: ubuntu-latest
```